CHANGELOG.md

```diff
@@ -1,6 +1,32 @@
 Changelog
 =========
 
+[1.3.5] - 2022-07-19
+--------------------
+
+### New Features
+
+- none
+
+### Bug Fixes
+
+- none
+
+### Other Changes
+
+- gather network facts default_ipv4 default_ipv6 (#56)
+
+Gather fact subset `network` instead of `min`
+Ensure facts `default_ipv4` and `default_ipv6`
+Ensure all tests work when using ANSIBLE_GATHERING=explicit
```

Comment:
Perhaps better use a sublist for those items? Currently they are rendered like [this](https://github.com/linux-system-roles/vpn/blob/140153e074331076f3ce5717814eb521f6266b40/CHANGELOG.md).